### PR TITLE
Mention the initial pre-paid credit in the Quickstart guide

### PR DESCRIPTION
**Pull request by carolstran**
### Description

Growth squad has a ticket about communicating the initial 10 $/€ of pre-paid credit 🤑  This number is consistent for all new accounts, so it makes sense to add it to the documentation. I used the emnify Terms of Service to determine specifics about that amount. 

### Additional Context

Internal ticket 🎟️  [PGR-203](https://emnify.atlassian.net/browse/PGR-203)

[PGR-203]: https://emnify.atlassian.net/browse/PGR-203?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/quickstart/getting-started.md [hunk 1]
```diff
@@ -39,11 +39,16 @@ To begin developing your IoT solutions with emnify, you'll need an emnify accoun
 
 ![Claim your free evaluation package with 3 triple-cut SIMs to kickstart your well-managed IoT journey. Includes: Free 3 triple-cut SIMs, form factor (Nano, Micro, Mini), our full coverage, and global fast shipping.](assets/portal-order-evaluation-pack.png)
 
-You can order your free Evaluation SIM package on the emnify Portal with which you can test all emnify services.
-Currently, the 3 triple-cut SIM cards can be delivered to most destinations worldwide with express shipment.
+You can order your free Evaluation SIM package on the emnify Portal.
+The package also includes an initial 10 EUR or USD of pre-paid credit to help you test emnify's services.
```
Comment (techwriter-dev) [inline docs/quickstart/getting-started.md:43]: Actually, I question the need for including "prepaid" at all.
Grammarly is fine with this:

> This package includes a 10 EUR/USD credit for testing emnify's services.

```suggestion
This package includes a prepaid credit of 10 EUR/USD for testing emnify's services.
```

File: docs/quickstart/getting-started.md [hunk 2]
```diff
@@ -39,11 +39,16 @@ To begin developing your IoT solutions with emnify, you'll need an emnify accoun
...
+
+:::note
+You have 60 days to use this pre-paid credit.
+:::
 
+Currently, the 3 triple-cut SIM cards can be delivered to most destinations worldwide with express shipment.
 
-[Log into your emnify account](https://portal.emnify.com/sign) and follow these steps:
+To get your free Evaluation SIM package, [log into your emnify account](https://portal.emnify.com/sign) and follow these steps:
```
Comment (techwriter-dev) [inline docs/quickstart/getting-started.md:51]:
```suggestion
To get your free Evaluation SIM package, [log in to your emnify account](https://portal.emnify.com/sign) and follow these steps:
```
**login** (noun) vs **log in** (verb) thus  [... log in to ...](https://www.express.co.uk/life-style/science-technology/996184/Gmail-sign-in-how-to-log-in-Gmail-account-easy-create-Gmail-email).

**Review by techwriter-dev — CHANGES_REQUESTED** [review]
A minor change to make things more concise and a minor correction.

**Comment by carolstran** [comment]
@techwriter-dev Oh sorry! I didn't ping you for a review yet because it still needs to be reviewed by Bas. Next time I'll make sure to add a note, but in the meantime, I can apply your suggestions 😊

File: docs/quickstart/getting-started.md (hunk 2 above)
Comment (techwriter-dev) [reply docs/quickstart/getting-started.md:51]: This is still not resolved.

Correct: *log* **in to** *your emnify account*

Perhaps these [Quora answers](https://www.quora.com/Which-one-is-correct-Log-in-to-your-account-Login-into-your-account-Log-in-your-account-or-Log-into-your-account-Ive-seen-people-using-all-of-them-and-Im-a-bit-confused) can explain it more clearly.

**Review by techwriter-dev — CHANGES_REQUESTED** [review]
Some corrections are still needed.

**Comment by carolstran** [comment]
@techwriter-dev My bad, thought I batched before 🤦🏼‍♀️  Should be ready now!

**Review by techwriter-dev — APPROVED** [review]
Perfect!

**Pull request merged**